circe-akka-serializer/src/main/scala/org/virtuslab/ash/CirceAkkaSerializer.scala

```diff
@@ -116,11 +116,16 @@ abstract class CirceAkkaSerializer[Ser <: AnyRef: ClassTag](system: ExtendedActo
     val foundSerializables = reflections.getSubTypesOf(classTag[Ser].runtimeClass).asScala.filterNot(_.isInterface)
     foundSerializables.foreach { clazz =>
       try {
-        codecsMap.get(parents(clazz.getName))
+        val codec = codecsMap(parents(clazz.getName))
+        assert(codec._1 != null)
```

Comment:
It's impossible to check in compile-time

Reply:
Ok! pls document (in Scaladoc?) what are the specific setups that lead to this assertion failing & how to mitigate/fix such errors 🙏🏻

Reply:
I believe this is covered by an error message:
```
"Codec for [{}] is null. If this codec is custom defined, use def or lazy val instead of val."
```